src/System.Collections/ref/System.Collections.cs

```diff
@@ -565,7 +565,7 @@ public partial class SortedList<TKey, TValue> : System.Collections.Generic.IColl
         void System.Collections.Generic.ICollection<System.Collections.Generic.KeyValuePair<TKey, TValue>>.CopyTo(System.Collections.Generic.KeyValuePair<TKey, TValue>[] array, int arrayIndex) { }
         bool System.Collections.Generic.ICollection<System.Collections.Generic.KeyValuePair<TKey, TValue>>.Remove(System.Collections.Generic.KeyValuePair<TKey, TValue> keyValuePair) { throw null; }
         System.Collections.Generic.IEnumerator<System.Collections.Generic.KeyValuePair<TKey, TValue>> System.Collections.Generic.IEnumerable<System.Collections.Generic.KeyValuePair<TKey, TValue>>.GetEnumerator() { throw null; }
-        void System.Collections.ICollection.CopyTo(System.Array array, int index) { }
+        void System.Collections.ICollection.CopyTo(System.Array array, int arrayIndex) { }
```

Comment:
In what situation does this parameter name on an explicit interface implementation matter? i.e. when will a developer ever see this?

Reply:
Reflection scenarios?

Reply:
> reflection

Hmm, I guess so.  Ok.